app/init/database/filters.php

```diff
@@ -77,13 +77,18 @@ function (mixed $value, Document $document, Database $database) {
         ]);
 
         foreach ($attributes as $attribute) {
-            if ($attribute->getAttribute('type') === Database::VAR_RELATIONSHIP) {
+            $attributeType = $attribute->getAttribute('type');
+            if ($attributeType === Database::VAR_RELATIONSHIP) {
                 $options = $attribute->getAttribute('options');
                 foreach ($options as $key => $value) {
                     $attribute->setAttribute($key, $value);
                 }
                 $attribute->removeAttribute('options');
             }
+            if ($attributeType === Database::VAR_STRING) {
+                $filters = $attribute->getAttribute('filters', []);
+                $attribute->setAttribute('encrypt', in_array('encrypt', $filters));
+            }
```

Comment:
In these sort of cases, lets always use a `switch`. Even if we only have two cases now, switch is easier to extend cleanly in future than multiple if's